custom_components/tuya_local/devices/solar_inverter_stepuptech.yaml

```diff
@@ -0,0 +1,135 @@
+name: Micro Inverter Stepuptech SP800
+products:
+  - id: r73jxzrtcda4uwwb
+    name: SP800
+primary_entity:
+  entity: sensor
+  name: Daily energy out
+  class: energy
+  dps:
+    - id: 107
+      type: integer
+      name: sensor
+      unit: kWh
+      class: total_increasing
+      force: true
+      mapping:
+        - scale: 100
+secondary_entities:
+  - entity: sensor
+    name: PV power
+    class: power
+    category: diagnostic
+    dps:
+      - id: 101
+        type: integer
+        name: sensor
+        unit: W
+        force: true
+        class: measurement
+        mapping:
+          - scale: 100
+  - entity: sensor
+    name: PV voltage
+    class: voltage
+    category: diagnostic
+    dps:
+      - id: 103
+        type: integer
+        name: sensor
+        unit: V
+        force: true
+        mapping:
+          - scale: 100
+  - entity: sensor
+    class: temperature
+    dps:
+      - id: 104
+        type: integer
+        name: sensor
+        unit: C
+        force: true
+        class: measurement
+        mapping:
+          - scale: 10
+  - entity: sensor
+    name: AC current
+    class: current
+    category: diagnostic
+    dps:
+      - id: 106
+        type: integer
+        name: sensor
+        unit: A
+        force: true
+        mapping:
+          - scale: 100
+  - entity: switch
+    name: Inverter switch
+    category: config
+    dps:
+      - id: 108
+        type: boolean
+        name: switch
+        optional: true
+  - entity: sensor
+    name: AC power
+    class: power
+    category: diagnostic
+    dps:
+      - id: 109
+        type: integer
+        name: sensor
+        unit: W
+        force: true
+        class: measurement
+        mapping:
+          - scale: 100
+  - entity: sensor
+    name: power adjust
+    class: power
```

Comment:
class power should have a unit of W, or similar, not %.  What actually is this measuring? Power factor?

Reply:
It is to limit the power output of the inverter. 
This is a setting and I may have not choose the good syntax.
Per default, the value is 100%, but you can reduce the output power.